Source/engine.cpp

```diff
@@ -2690,6 +2713,13 @@ void Cl2Blit(BYTE *pDecodeTo, BYTE *pRLEBytes, int nDataSize, int nWidth)
 }
 
 /**
+ * @brief Blit a solid colder shape one pix larger then the given sprite shape, to the back buffer at the given coordianates
+ * @param col Colro index from current palette

```

Comment:
what's your favorite colro? mine is purple :p

Reply:
mine is potato